ada/testsuite/tests/properties/fully_qualified_name_2/test.py

```diff
@@ -0,0 +1,21 @@
+from __future__ import absolute_import, division, print_function
```

Comment:
Isn't this `test.py` useless if we use `driver: inline-playground`?

Reply:
It is! Good catch :)